packages/gui/src/types/css.ts

```diff
@@ -35,6 +35,7 @@ export const enum PercentageLengthUnits {
 }
 export const enum UnitlessUnits {
   Number = 'number',
+  Calc = 'calc',
```

Comment:
Debated whether `calc` is actually a unit for a little. Open to other opinions here

Reply:
Yeah that's a tricky one, think for now we can leave it here. Can always break it out when we start to support more functions.